### PR TITLE
Initial documentation

### PR DESCRIPTION
**Pull request by albinahlback**
Initial documentation.

### CODE REVIEW

**Comment by albinahlback** [comment]
Not sure why these tests fail, do you know @kalmarek?

File: docs/src/group_elements.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+```@docs
+one(::GroupElement)
+isequal(::GroupElement, ::GroupElement)
+:(^)(::GroupElement, ::Integer)
+:(/)(::GroupElement, ::GroupElement)
+order(::Type{<:Integer}, ::GroupElement)
+conj(::GroupElement, ::GroupElement)
+:(^)(::GroupElement, ::GroupElement)
+commutator(::GroupElement, ::GroupElement)
```
Comment (albinahlback) [inline docs/src/group_elements.md:43]: Fix

**Comment by albinahlback** [comment]
I added a prefix of `GroupsCore` to the calling of the docstring for `parent_type`, since it is not (at the moment) exported.

I have no comments besides that. Great work, @kalmarek!

**Comment by kalmarek** [comment]
:tada:

**Pull request merged**